js/geo/lng_lat.js

```diff
@@ -18,6 +18,9 @@ var wrap = require('../util/util').wrap;
  * @param {number} lat Latitude, measured in degrees.
  * @example
  * var ll = new mapboxgl.LngLat(-73.9749, 40.7736);
+ * @see {@link https://www.mapbox.com/mapbox-gl-js/example/mouse-position/|Get coordinates of the mouse pointer}
```

Comment:
We can simplify links to relative URLs like `/mapbox-gl-js/example/mouse-position/`. However it looks, it would be nice to get it working with `localhost:4000`.